torchbenchmark/models/clip/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/facebookresearch/multimodal.git
```

Comment:
We suggest to pin the version of this dependency with commit hash, because when we are benchmarking pytorch nightly, and changes from upstream model code might cause false positive performance signals.